docs/src/content/docs/getting-started/configuration.mdx

```diff
@@ -38,69 +45,88 @@ Here's an example `access_control` configuration:
+Here's an example `apply_requirements` configuration with explanations:
 
 ```yaml
 apply_requirements:
+  # Creates a status check that prevents merging until apply is complete
   create_pending_apply_check: true
+  
   checks:
-    - tag_query: ""
+    - tag_query: ""  # Apply to all workspaces
+      
+      # Requires PR approvals
       approved:
-        enabled: true
-        any_of: []
-        any_of_count: 2
-        all_of: []
+        enabled: true        # Enable approval checking
+        any_of: []           # No specific approvers required
+        any_of_count: 2      # Requires at least two approvals
+        all_of: []           # No mandatory approvers
+      
+      # Prevents apply if merge conflicts exist
       merge_conflicts:
         enabled: true
+      
+      # Requires all status checks to pass
       status_checks:
-        enabled: true
-        ignore_matching:
-          - "ci/.*"
+        enabled: true              # Enable status check verification
+        ignore_matching:           # Regex patterns for checks to ignore
+          - "ci/.*"                # Ignores all CI status checks
 ```
 
-This configuration requires that the pull request has at least 2 approvals, no merge conflicts, and all status checks (except those matching `ci/.*`) have passed before an apply can be performed.
+This configuration requires that the pull request has at least two approvals, no merge conflicts, and all status checks (except those matching `ci/.*`) have passed before an apply can be performed.
 
 When `create_pending_apply_check` is enabled, Terrateam will create a `Terrateam Apply` GitHub status check. Combined with [GitHub Rulesets](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-rulesets/about-rulesets), this prevents the pull request from being merged until all Apply operations have completed.
 
-For more information, see the [Apply Requirements](/configuration-reference/apply-requirements) section of the Configuration Reference.
+See [Apply Requirements](/configuration-reference/apply-requirements) for details.
 
 ## Dirs
 
-Dirs allow you to define which [Tags](/advanced-workflows/tags), Workspaces, and [When Modified](/configuration-reference/when-modified) rules rules apply to specific directories in your repository.
+Dirs allow you to define which [Tags](/advanced-workflows/tags), Workspaces, and [When Modified](/configuration-reference/when-modified) rules apply to specific directories in your repository.
 
 Here's an example `dirs` configuration:
 
 ```yaml
 dirs:
-  ec2:
-    tags: [aws, ec2]
+  ec2:  # Directory name (relative to repository root)
+    tags: [aws, ec2]  # Directory-level tags
     workspaces:
-      production:
-        tags: [production]
+      production:  # Workspace name
+        tags: [production]  # Workspace-specific tags
     when_modified:
       file_patterns: ["ec2/*.tf", "ec2/*.tfvars", "iam/*.tf", "iam/*.tfvars"]
```

Comment:
This should use `${DIR}/*.tf` syntax for the `ec2` ones.  And probably layered runs for `iam`.